### DISCUSSION TITLE
Adding Hardware Acceleration to a new install

### DISCUSSION THREAD

**Discussion by yourpalfranc** [blakeblackshear/frigate#13890] [Ask A Question] [answered]
On an i7-6700K I tried enabling hardware acceleration to a new install with 1 working camera by adding 

name_of_your_camera:
    ffmpeg:
      inputs: ...
      hwaccel_args: preset-vaapi

to my config. When I restarted Frigate, my working camera didn't display an image.

I don't seem to be able to add another camera without some sort of graphics support, and so far, I've been unable to get my Coral to work.

I'm totally new to Docker, so to install hardware acceleration, do I edit the docker-compose.yml file and add the code from the Getting Started documentation??

version: "3.9"
services:
  frigate:
    ...
    devices:
      - /dev/dri/renderD128 # for intel hwaccel, needs to be updated for your hardware
    ...

Or, do I create a new docker-compose.yml file with only this code and then run docker compose up -d from the folder where docker-compose.yml resides?

**Comment by hawkeye217** [comment, marked as answer]
Yes, just edit your existing docker-compose file to add the `/dev/dri/renderD128` device, and then rerun `docker compose up -d`.

**Reply by yourpalfranc** [reply]
Seemed to have worked. No errors. I'll try adding more cameras later. Thank you!

**Reply by hawkeye217** [reply]
Great. Happy to help!
